src/wasm-lib/kcl/src/unparser.rs

```diff
@@ -167,11 +168,14 @@ impl VariableDeclaration {
     pub fn recast(&self, options: &FormatOptions, indentation_level: usize) -> String {
         let indentation = options.get_indentation(indentation_level);
         self.declarations.iter().fold(String::new(), |mut output, declaration| {
+            let keyword = match self.kind {
+                VariableKind::Fn => "fn ",
+                VariableKind::Let | VariableKind::Const | VariableKind::Var => "",
```

Comment:
PR description said the keyword was going to be preserved for now. So I guess update the description?